src/foundry/foundry.js/applications/formApplication.d.ts

```diff
@@ -45,7 +46,7 @@ declare abstract class FormApplication<
    * The values of this Array are inner-objects with references to the MCE editor and other metadata
    * @defaultValue `{}`
    */
-  editors: Partial<Record<string, FormApplication.FormApplicationEditor>>;
+  editors: Record<string, FormApplication.FormApplicationEditor>;
 
   /**
    * Assign the default options which are supported by the entity edit sheet.
```

Comment:
Please document the value returned (I think we do that with `@defaultValue` in a lot of other places for the `defaultOptions`